### PR TITLE
Feature/step4

### PR DESCRIPTION
**Pull request by 2003ma**
## CHECKS :warning:

Please make sure you are aware of the following.

- [ ] **The changes in this PR doesn't have private information

### CODE REVIEW

File: go/app/main.go [hunk 1]
```diff
@@ -151,19 +155,70 @@ func getItem(c echo.Context) error {
...
 	}
-	itemsData, err := readFile(c, "items.json")
+	db, err := sql.Open("sqlite3", "../db/mercari.sqlite3")
 	if err != nil {
-		return errMessage(c, err, http.StatusBadRequest, "Unable to open items.json")
+		return errMessage(c, err, http.StatusBadRequest, "Unable to open database")
+	}
+	defer db.Close()
+
+	stmt, err := db.Prepare("SELECT items.id,items.name,categories.name,items.image_name FROM items LEFT JOIN categories ON items.category=categories.id")
```
Comment (F-0728) [inline go/app/main.go:164]: ここ，毎回テーブルの名前打つの読みにくいと思ったら，省略形で書けたはず！
参考までに :pray:

https://oss-db.jp/dojo/dojo_13

File: go/app/main.go [hunk 2]
```diff
@@ -30,11 +31,23 @@ type ItemsData struct {
 }
 
 type Item struct {
```
Comment (mshibuya) [inline go/app/main.go:33]: `type Item` is unused, right?
Does that mean we don't need to differentiate `Item` and `ItemDisplay`?

File: db/items.sql [hunk 3]
```diff
@@ -0,0 +1 @@
+CREATE TABLE items(id INTEGER PRIMARY KEY,name TEXT,category INTEGER,image_name TEXT);
```
Comment (mshibuya) [inline db/items.sql:1]:
```suggestion
CREATE TABLE items(id INTEGER PRIMARY KEY,name TEXT,category_id INTEGER,image_name TEXT);
```

It's a common convention to name a foreign key as `..._id`.

**Review by mshibuya — COMMENTED** [review]
The implementation is nicely done! I've made some minor comments 👍

**Pull request merged**